test/test_statsd.js

```diff
@@ -487,6 +487,55 @@ function doTests(StatsD) {
 
   });
 
+  describe('#timer', function() {
+    it('should send stat and time to execute to timing function', function(finished) {
+      udpTest(function (message, server) {
+        // Search for a string similar to 'test:0.123|ms'
+        var re = RegExp("(test:)([0-9]+\.[0-9]+)\\|{1}(ms)");
```

Comment:
regex has to be used because the `value` sent will always change (since it's the time taken to execute the function`